jbmc/unit/java_bytecode/java_bytecode_parser/parse_java_attributes.cpp

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************\
+
+ Module: Unit tests for converting annotations
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#include <java-testing-utils/load_java_class.h>
+#include <java_bytecode/java_bytecode_convert_class.h>
+#include <java_bytecode/java_bytecode_parse_tree.h>
+#include <java_bytecode/java_types.h>
+#include <testing-utils/catch.hpp>
+
+// See
+// https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.7.16.1
+SCENARIO(
+  "java_bytecode_parse_attributes",
+  "[core][java_bytecode][java_bytecode_parser]")
+{
+  GIVEN("Some class files in the class path")
+  {
+    WHEN("Parsing the InnerClasses attribute for a public inner class")
+    {
+      const symbol_tablet &new_symbol_table =
+        load_java_class("InnerClasses", "./java_bytecode/java_bytecode_parser");
+
+      THEN("The class should be marked as public")
+      {
+        const symbolt &class_symbol =
+          *new_symbol_table.lookup("java::InnerClasses$PublicInnerClass");
+        const java_class_typet java_class =
+          to_java_class_type(class_symbol.type);
+        REQUIRE(java_class.get_inner_class() == ID_is_inner_class);
```

Comment:
I think this is wrong, I think the string will be something like "1" or "true" - which probably means if this passes CI these unit tests aren't being run.

Reply:
Indeed, it is the missing from Makefile as the CMake build fails:
https://travis-ci.org/diffblue/cbmc/jobs/397934975#L7792
`1 == is_inner_class`